Colander/Classes/DateExtensions.swift

```diff
@@ -3,32 +3,3 @@ import SwiftDate
 internal extension Calendar {
     static let gregorian = Calendar(identifier: Calendar.Identifier.gregorian)
 }
-
-internal extension Date {
```

Comment:
These were redundant with `SwiftDate` and enforced a calendar that was not necessarily that which was provided for use elsewhere.